clientv3/integration/metrics_test.go

```diff
@@ -41,12 +41,14 @@ func TestV3ClientMetrics(t *testing.T) {
 	var (
 		addr = "localhost:27989"
 		ln   net.Listener
-		err  error
+
 	)
 
 	// listen for all Prometheus metrics
 	donec := make(chan struct{})
 	go func() {
+		var err error
```

Comment:
No need to declare here. Just use ':=' on the first occurrence?

Reply:
the first occurrence is here: `ln, err = transport.NewUnixListener(addr)` but `ln` is shared between goroutines